assets/src/components/vehiclePropertiesPanel.tsx

```diff
@@ -86,7 +104,11 @@ const Header = ({
       <div className="m-vehicle-properties-panel__variant-name">
         {formatRouteVariant(vehicle)}
       </div>
-      <ScheduleAdherence vehicle={vehicle} />
+      {vehicle.headwaySpacing ? (
```

Comment:
does this need to be behind the flag as well?

Reply:
Good catch! Added that check.